src/renderers/shared/reconciler/ReactReconciler.js

```diff
@@ -40,13 +41,15 @@ var ReactReconciler = {
     transaction,
     nativeParent,
     nativeContainerInfo,
-    context
+    context,
+		nodesToReuse
```

Comment:
Even if it is an array, it represents a single RenderedComponent, correct?

I guess I don't feel strongly, just wanted to be sure, because nodes implies multiple rendered components (eg. all of a component's children) rather than a single rendered result.  But I understand where you're coming from, so I'll leave it to be your call.